test_problems/surfSolverTest/surfaceSolver.cpp

```diff
@@ -6,12 +6,6 @@
 // This file is part of Cantera. See License.txt in the top-level directory or
 // at https://cantera.org/license.txt for license and copyright information.
 
-//  Example
-//
-//  Read a surface growth mechanism and calculate the solution
```

Comment:
While the comments at the top were certainly not extremely helpful, why remove them?

Reply:
The original comment was "Read a surface growth mechanism and calculate the solution using Placid". I don't know what "Placid" is, so my assumption is that this comment is just copy-pasta garbage and that's why I removed it.

Reply:
I read that :laughing: ... but you probably know what this test actually does. Apart from that, all others are left with is purely based on the name `surfSolverTest/surfaceSolver.cpp `

Reply:
Closing.